Content.Server/Atmos/EntitySystems/AtmosphereSystem.cs

```diff
@@ -107,7 +107,7 @@ public override void Update(float frameTime)
         var query = EntityQueryEnumerator<AtmosExposedComponent, TransformComponent>();
         while (query.MoveNext(out var uid, out _, out var transform))
         {
-            var air = GetContainingMixture((uid, transform));
+            var air = GetContainingMixture((uid, transform), false, true);
```

Comment:
This would need to be in a separate PR if we make this change. I need to spend some time looking at this and see if exciting the mixture is really needed to get temperature to transfer correctly with `AtmosExposed`, but this may have significant detrimental performance impacts.

Reply:
As a nitpick, please name the optional non-default arguments (e.g. `GetContainingMixture((uid, transform), excite: true)`. This makes it a lot more apparent what the argument is, especially if someone is reviewing from a diff/GitHub/anything but Rider.

Reply:
When I tested it without exciting it did work, but then the room's gas mix could have temperature gradients that wouldn't change until something else excited the gas in the room.

Reply:
> This would need to be in a separate PR if we make this change.

In the interest of fixing the bug, Ill move the gas emission stuff to another PR, I think for it to work I need burning to have temperature behave reasonably so it makes more sense to start there. I do agree that its kind of soaped into something else in supporting making static fires modeled now.